app/controllers/generic_object_definition_controller.rb

```diff
@@ -23,6 +24,30 @@ def self.model
     GenericObjectDefinition
   end
 
+  def button
+    javascript_redirect(
+      case params[:pressed]
+        when 'generic_object_definition_new'
```

Comment:
also, rubocop is right, `when` should be on the same level as `case` in ruby

Reply:
True

Reply:
Fixed in 8e2993b